p2p/transport/webrtc/stream_write.go

```diff
@@ -110,11 +110,19 @@ func (s *stream) SetWriteDeadline(t time.Time) error {
 	return nil
 }
 
+func (s *stream) sendBufferSize() int {
```

Comment:
moving comments
```suggestion
// sendBufferSize() is the maximum data we enqueue on the underlying data channel for writes.
// The underlying SCTP layer has an unbounded buffer for writes. We limit the amount enqueued
// per stream is limited to avoid a single stream monopolizing the entire connection.
func (s *stream) sendBufferSize() int {
```